redstone/client.py

```diff
@@ -418,6 +418,14 @@ def create(self, name, payload=None, raw_payload=None, root=False):
         elif payload is not None:
             data['resources'][0]['payload'] = base64.b64encode(payload).decode('utf-8')
 
+        # TKEK related fields
+        if encryption is not None:
+            data['resources'][0]['encryptionAlgorithm'] = 'RSAES_OAEP_SHA_256'
```

Comment:
No point in adding a keyword arg if you are just going to set this to a hardcoded value.

Put this set into the if block where you check if they passed a signed_nonce or iv